JSDemos/Demos/TabPanel/Overview/Angular/app/app.component.css

```diff
@@ -1,45 +1,122 @@
-::ng-deep .tabpanel-item {
-  height: 200px;
-  -webkit-touch-callout: none;
-  -webkit-user-select: none;
-  -khtml-user-select: none;
-  -moz-user-select: none;
-  -ms-user-select: none;
-  user-select: none;
-  padding-left: 25px;
-  padding-top: 55px;
+::ng-deep .tabpanel-demo {
+  display: flex;
+  height: 100%;
+  min-height: 450px;
 }
 
-::ng-deep .mobile .tabpanel-item {
-  padding-top: 10px;
+::ng-deep .widget-container {
+  display: inline;
+  flex-grow: 1;
+  min-width: 450px;
+  padding: 16px 32px;
 }
 
-::ng-deep .tabpanel-item > div {
-  float: left;
-  padding: 0 85px 10px 10px;
+::ng-deep .dx-theme-material .dx-tabpanel {
+  border-radius: 8px;
+  overflow: hidden;
 }
 
-::ng-deep .tabpanel-item p {
-  font-size: 16px;
-  margin: 0;
+::ng-deep .dx-tabs-vertical {
+  min-width: 120px;
 }
 
-::ng-deep .item-box {
-  font-size: 16px;
-  margin: 15px 0 45px 10px;
+::ng-deep .dx-multiview-item-content {
+  padding: 24px;
 }
 
 ::ng-deep .options {
+  display: inline-flex;
+  flex-direction: column;
+  flex-shrink: 0;
+  box-sizing: border-box;
+  width: 300px;
   padding: 20px;
   background-color: rgba(191, 191, 191, 0.15);
-  margin-top: 20px;
+}
+
+::ng-deep .tabpanel-item .task-item:not(:last-child) {
+  margin-bottom: 12px;
 }
 
 ::ng-deep .caption {
-  font-size: 18px;
   font-weight: 500;
+  font-size: 18px;
 }
 
 ::ng-deep .option {
-  margin-top: 10px;
+  margin-top: 20px;
+}
+
+::ng-deep .task-item {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  height: 48px;
+  padding: 8px;
+  border-radius: 4px;
+  box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.15);
+}
+
+::ng-deep .task-item::before {
+  content: '';
+  position: absolute;
+  top: 8px;
+  left: 6px;
+  bottom: 8px;
+  width: 3px;
+  border-radius: 4px;
+}
+
+::ng-deep .task-item-priority-hight::before {
```

Comment:
I think this is a typo - 'hight'. I think 'high' should be.

Reply:
[b47c4be5bc56d7d1d81ce5aa0c922116cd6b34b7]